HanksMineralEmporium/Service/AuthenticationService/ AuthenticationService.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+using System.Web;
+using HanksMineralEmporium.Core.UserManagement;
+using HanksMineralEmporium.Core.UserManagement.Exception;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.DependencyInjection;
+using System.Threading.Tasks;
+using System.Runtime.CompilerServices;
+using MudBlazor;
+using Microsoft.AspNetCore.Http.HttpResults;
+using Microsoft.AspNetCore.Http;
+using HanksMineralEmporium.Service.AuthenticationService.Exception;
+/// <summary>
```

Comment:
Move this doc string to above AuthenticationService instead of above the namespace. Furthermore, you can remove all of the stuff under \</summary>